Dockerfile

```diff
@@ -120,4 +128,4 @@ ENV DJANGO_SETTINGS_MODULE storage_service.settings.production
 
 EXPOSE 8000
 
-ENTRYPOINT pyenv exec python${PYTHON_VERSION} -m gunicorn --config=/etc/archivematica/storage-service.gunicorn-config.py storage_service.wsgi:application
+ENTRYPOINT pyenv exec python${PYTHON_VERSION} -m gunicorn --config=/etc/archivematica/storage-service.gunicorn-config.py storage_service.wsgi:application
```

Comment:
Is this change necessary?

Reply:
Not really, it seems editor issue.

Reply:
Was this change added to the `qa/0.x` branch?

Reply:
Was it intended? I understood the change was unnecessary.

Reply:
Not really.

Reply:
Cool. No problem, but I suggest you before requesting reviews or merging PRs to always check the `Files changed` tab of your pull requests. It should only contain your *intended* changes and no more.

Reply:
I think for this specific case we could use the [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#end-of-file-fixer) from `pre-commit`.

Reply:
sure and thanks for the suggestions and notes.